snapshots/snapshot_generator/src/test/scala/uk/ac/wellcome/platform/snapshot_generator/SnapshotGeneratorFeatureTest.scala

```diff
@@ -81,14 +87,22 @@ class SnapshotGeneratorFeatureTest
               assertJsonStringsAreEqual(actualLine, expectedLine)
           }
 
-          val expectedJob = CompletedSnapshotJob(
-            snapshotJob = snapshotJob,
-            targetLocation =
-              s"http://localhost:33333/${s3Location.bucket}/${s3Location.key}"
-          )
+          val result = messageSender.getMessages[CompletedSnapshotJob].head
+
+          result.snapshotJob shouldBe snapshotJob
+
+          result.snapshotResult.indexName shouldBe worksIndex.name
+          result.snapshotResult.documentCount shouldBe works.length
+          result.snapshotResult.displayModel shouldBe Work.getClass.getCanonicalName
```

Comment:
I would hard code this, so we have some visibility of what this actually returns.